doc/whats-new.md

```diff
@@ -1,144 +1,215 @@
-  the deprecated XML exporter.
-- Some preliminary work for the Python 3 installers has been done.
+We are visualizing the `calculation_mode` field in the WebUI now,
+instead of the `job_type` field, which is less specific.
+
+We removed an excessive check from the WebUI: now if an output exists,
+it can be downloaded even if the full calculation was not successful.
+
+Now we show the user the error message in the case of a calculation
+that cannot be deleted from the WebUI.
+
+We changed the .npz exporters of the hazard outputs to make the import
+into a single QGIS layer easier.
+
+We now have in place an automatic testing mechanism making sure that
+all the outputs of the demos that are loadable by the QGIS plugin are
+actually imported and visualized correctly. The plugin itself has several
+new features, documented [here](https://plugins.qgis.org/plugins/svir/version/2.6.1/).
+
+We started work to make the risk outputs readable from the QGIS
+plugin and we will be able to plot some of them in the next release of
+the plugin.
+
+oq commands
+---------------
+
+We added four new `oq` commands:
+
+- `oq checksum <job_file_or_job_id>` computes a 32 bit checksum for a
+   calculation or a set of input files; this is useful to check if a
+   given calculation has been really performed with the given input files
+   and no parameters have been modified later;
+- `oq extract <what> <calc_id>` allows to save a specified output into an
+  .hdf5 file
+- `oq dump <dump.zip>` allows to dump the database of the engine and all
+  the datastores into a single .zip file
+- `oq restore <dump.zip> <oqdata>` allows to restore a dump into an empty
+  directory.
+
+We added a view `oq show dupl_sources` and enhanced `oq info
+job.ini` to display information about duplicated sources; moreover,
+some warnings are logged if duplicated sources are found. In the
+future we may decide to optimize the engine for this situation; for
+the moment (as in the past) the engine is performing twice the same
+computation if the same source appears twice in the full source model.
+
+Moreover we extended the `oq reset` command to work on multi user
+installations. Finally, we renamed `oq engine --version-db` to `oq engine
+--db-version`, to avoid confusion between `oq --version` (the version
+of the code) and `oq engine --version` (that was returning the version
+of the database).
+
+Other
+-----
+
+We changed the behaviour of 'CTRL-C' to make sure that all children
+processes are killed when a calculation is interrupted.
+
+Python 3.6 is not officially supported, however we have fixed the only test
+which was breaking and we know for sure that it works. We also fixed some
+tests breaking with numpy 1.13 which however is still not officially supported.
+
+We fixed some tests breaking on macOS; the numbers there are slightly
+different, since the scientific libraries are compiled differently.
+This platform should be trusted less than Linux and Windows.
```

Comment:
I would skip this statement. Or at least just say 'This platform should be trusted less than Linux, which remains the reference'.